tldr/semantic_old.py

```
@@ -0,0 +1,1062 @@
+        respect_ignore: If True, respect .tldrignore patterns (default True).
+
+    Returns:
+        List of EmbeddingUnit objects with enriched metadata.
+    """
+    from tldr.api import get_code_structure, build_project_call_graph, get_imports
+    from tldr.tldrignore import load_ignore_patterns, should_ignore
+
+    project = Path(project_path).resolve()
+    units = []
+
+    from tldr.tldrignore import load_ignore_patterns
```

Ruff (0.14.10)

Redefinition of unused `load_ignore_patterns` from line 270

Remove definition: `load_ignore_patterns`

(F811)